CMakeLists.txt

```diff
@@ -49,6 +49,10 @@ option(INSTALL_VENDORED_LIBS
     "Install vendored thirdparty libraries along with liborc"
     ON)
 
+option(ENABLE_CPP_WARNINGS
```

Comment:
Actually, I'd propose that we make it: STOP_BUILD_ON_WARNING
and default it to ON

Reply:
Agreed. I'll post an update tomorrow.

Reply:
Done.